src/hotspot/cpu/riscv/vm_version_riscv.hpp

```diff
@@ -297,6 +297,13 @@ class VM_Version : public Abstract_VM_Version {
   // RISCV64 supports fast class initialization checks
   static bool supports_fast_class_init_checks() { return true; }
   static bool supports_fencei_barrier() { return ext_Zifencei.enabled(); }
+
+  static bool supports_float16_float_conversion() {
```

Comment:
Is it safe to simply rename this as `supports_float16` like other CPUs? I suppose it will still work in functionality even if we only have the `Zfhmin` extension, right?

Reply:
In fact, I think `supports_float16` is confusing, in particular on riscv, as e.g. `AddHF` requires `UseZfh`, but `ConvF2HF` requires `UseZfh || UseZfhmin`, so I think it's better to keep supports_float16_float_conversion, as its naming is much more explicit and clearer.

Reply:
All right!